tests/test_sink.py

```diff
@@ -1419,11 +1419,11 @@ async def test_cancel_sync(extractor_task_done, sink_task_done, force_cancel):
     es._sink = Mock()
     es._sink.force_cancel = Mock()
 
-    es._extractor_task = Mock()
+    es._extractor_task = mock.create_autospec(asyncio.Task)
```

Comment:
TIL about `create_autospec`, neat!

Reply:
Same. I had to google for it, because I figured there had to be a way to use mocks but still be protected against AttributeError issues like this. I'll have to use these more going forward.